scripts/rte_ctrl

```diff
@@ -102,4 +102,5 @@ case "$CMD" in
     echo "Invalid command: \"$1\""
     echo ""
     usage
+    exit -1
```

Comment:
Isn't it more common for the exit code in scripts to be positive number?

Reply:
Yes, changed.